### DISCUSSION TITLE
Can you adjust formatting of markdown checklist spacing?

### DISCUSSION THREAD

**Discussion by tspng** [zed-industries/zed#10545] [Feature Requests]
Hi
Is there a way to configure how markdown is formatted?

I would like to use just one space between a hyphen and the square brackets in checklists. 
The formatter also inserts newlines for sub-items if there are any other newlines in the item list.

input (which I would like to stay that way):
```md
checklist test

- [x] item1
- [ ] item2
    - [ ] item3
    - [ ] item4

- [ ] item5
```

formatted version:
```md
checklist test

-   [x] item1
-   [ ] item2

    -   [ ] item3
    -   [ ] item4

-   [ ] item5
```

**Comment by xpe** [comment]
@tspng Have you checked the Zed source code to see what formatter is used?

**Reply by SomeoneToIgnore** [reply]
Markdown is formatted by prettier by default, there's no LSP (hence, LSP formatting) for this language.
I have not found anything related to markdown settings in https://prettier.io/docs/en/options

**Comment by xpe** [comment]
I used the example posted by @tspng and formatted it with Zed Preview 0.137.0. Here is what I get:

```md
checklist test

- [x] item1
- [ ] item2

  - [ ] item3
  - [ ] item4

- [ ] item5
```

I see the (vertical) line between item2 and item3. But I don't see any additional horizontal space between "-" and "[]" as shown in the example above.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
